README.md

```diff
@@ -248,11 +248,11 @@ For Mac, there is only 64-bit distribution.
 
 Here is an example script to build with the static library:
 
-`g++ main.cpp -Icpp/Mac_64/hazelcast/external/include -Icpp/Mac_64/hazelcast/include cpp/Mac_64/hazelcast/lib/libHazelcast3.10.1_64.a`
```

Comment:
cpp/Mac_64/hazelcast/external and cpp/Mac_64/external are the same locations. `cpp/Mac_64/hazelcast/external` is just a symbolic link to `cpp/Mac_64/external`. Did it not work for you?